### PR TITLE
Dockerfile.okd: Use shaded GCS jar in OKD image

### PR DESCRIPTION
**Pull request by chancez**

### CODE REVIEW

**Comment by chancez** [comment]
/hold
waiting on https://github.com/openshift/release/pull/4656 before merging so that all our images for the 4.2 stream are using the correct CI configs.

**Review by timflannagan — APPROVED** [review]
/lgtm

**Pull request merged**
